### ISSUE TITLE
can't index documents after setting custom primaryKey and using Laravel Scout extended aggregator

### ISSUE THREAD

**Issue by goellner** [meilisearch/meilisearch#5012]
**Describe the bug**
After changing the `primaryKey` of an index and trying to add a document to it, it throws `index_primary_key_already_exists`. The index has the correct new `primaryKey` and the pushed document also includes the defined key. 

**To Reproduce**
$client->deleteIndex('product_categories_frontend');
$client->createIndex('product_categories_frontend');
$client->updateIndex('product_categories_frontend', ['primaryKey' => 'unique_id']);

then index a document with `unique_id` set.

**Expected behavior**
it should index the document

**Meilisearch version:**
v1.10.1

**Comment by curquiza** [comment]
Hello @goellner 
I did not manage to reproduce your issue by doing exactly your steps

Can you send cURL commands your applied to get your error?

**Comment by goellner** [comment]
I just figured out I only have this, when indexing with Laravel Scout. Still investigating if this only happens when using Aggregors (https://github.com/algolia/scout-extended). 

Tried recreating with cURL commands and it worked, converted the `toSearchableArray()` result to json. Now have to test if the scout package is at fault or the scout-extended. having a hard time finding a place where i can dump the body and path of the request to meilisearch in those composer packages.
